projects/project_ideas.md

```diff
@@ -0,0 +1,59 @@
+# CS 7290: Project Ideas
+**Prof. Robert Ness**
+
+## 1. State-of-the-art Causal Inference methods
+
+You are encouraged to explore Causal Inference literature and to explore the papers that suits your research interests. For reference, take a look at the proceedings of NeurIPS and UAI for last four years. The applications of these methods do not limit to AI and computer science. They expand to Epidemiology, Social sciences, Medicines, Statistics and Political sciences. You can pick the case studies that suit your interest.
```

Comment:
Let's change this to implementing a method from one paper, and providing code.  They would have to source their own dataset.  And I think it should indeed be limited to AI and machine learning, because that is the theme of the course.